src/tasks/AotCompilerTask/MonoAOTCompiler.cs

```diff
@@ -121,6 +121,11 @@ public class MonoAOTCompiler : Microsoft.Build.Utilities.Task
     /// </summary>
     public string? NetTracePath { get; set; }
 
+    /// <summary>
+    /// Directory containing all assemblies referenced in a .nettrace collected from a separate device needed by dotnet-pgo. Necessary for mobile platforms.
+    /// </summary>
+    public ITaskItem[]? ReferenceAssembliesForPGO { get; set; }
```

Comment:
```suggestion
    public ITaskItem[] ReferenceAssembliesForPGO { get; set; } = Array.Empty<ITaskItem>();
```

.. then the property doesn't need to be nullable.